### PR TITLE
Add nodenetworkconfig for test cluster to connect to NESE

### PR DESCRIPTION
**Pull request by dystewart**
Addresses: [https://github.com/OCP-on-NERC/operations/issues/92](https://github.com/OCP-on-NERC/operations/issues/92)

### CODE REVIEW

File: cluster-scope/overlays/nerc-ocp-test/nodenetworkconfigurationpolicies/vlan-2173-nese.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    - name: bond0.2173
+      type: vlan
+      state: up
+      ipv4:
+        enabled: true
+        dhcp: true
+        auto-routes: true
+      vlan:
+        base-iface: bond0
+        id: 2173
```
Comment (naved001) [inline cluster-scope/overlays/nerc-ocp-test/nodenetworkconfigurationpolicies/vlan-2173-nese.yaml]: is it confirmed that this is, in fact, the right VLAN to access the ceph pool for the test cluster?

2177 is for nerc-ocp-infra
2173 is for nerc-ocp-prod

Comment (naved001) [reply cluster-scope/overlays/nerc-ocp-test/nodenetworkconfigurationpolicies/vlan-2173-nese.yaml]: @jtriley ^^Could you confirm the VLAN?

Comment (jtriley) [reply cluster-scope/overlays/nerc-ocp-test/nodenetworkconfigurationpolicies/vlan-2173-nese.yaml]: Should be 2175 for the test cluster

Comment (dystewart) [reply cluster-scope/overlays/nerc-ocp-test/nodenetworkconfigurationpolicies/vlan-2173-nese.yaml]: Looks like the interface is 2175: [https://massopencloud.slack.com/archives/C027TDE52TZ/p1678481163497359?thread_ts=1676390735.116989&cid=C027TDE52TZ](https://massopencloud.slack.com/archives/C027TDE52TZ/p1678481163497359?thread_ts=1676390735.116989&cid=C027TDE52TZ). Will update when confirmed

**Review by jtriley — APPROVED** [review]

**Pull request merged**